flask_monitoringdashboard/core/user_exception_logger.py

```diff
@@ -0,0 +1,13 @@
+from types import TracebackType
+from typing import Union
+
+ExcInfo = tuple[type[BaseException], BaseException, TracebackType]
+
+class ScopedExceptionLogger():
```

Comment:
why the mismatch between the filename and the class name? 
If this represents a `UserCapturedException` then I think that's a better name than Scoped? Otherwise, add a small comment to explain the meaning of `scoped`.